cheetah/accelerator.py

```diff
@@ -46,6 +46,11 @@ class Element:
     is_skippable = True
 
     def __init__(self, name=None, device="auto"):
+        """
+        Used to define parameters. When defined they can be accessed through self.parameter via other function.
+        The function parameters change for each class. Since the parameters of this funvtion are always equivalent to the ones of the class
+        there will be no further description of the parameters.
+        """
```

Comment:
Don't document `__init__`. Basically everything that function does should be obvious by the definition of the class already. Take a look at the [Stable Baselines3](https://github.com/DLR-RM/stable-baselines3) repository for a good example of how to do this well.